php/libraries/User.class.inc

```diff
@@ -11,6 +11,8 @@
  * @link     https://www.github.com/aces/Loris-Trunk/
  */
 
+use Http\Adapter\Guzzle6\Client as GuzzleAdapter;
+use GuzzleHttp\Psr7\Request;
```

Comment:
This should be using PSR-18, not a specific hardcoded implementation of PSR-18.

(To do that you'll probably need to wrap the reference to Guzzle in a factory somewhere.)

Reply:
@driusan I just sent a commit that I think does what you asked. Let me know if there's something to correct.